src/init.cpp

```diff
@@ -746,6 +747,7 @@ bool AppInit2(Config &config, boost::thread_group &threadGroup, CScheduler &sche
         LOGA("Prune configured to target %uMiB on disk for block and undo files.\n", nPruneTarget / 1024 / 1024);
         fPruneMode = true;
     }
+    pruneInterval = GetArg("-pruneinterval", DEFAULT_PRUNE_INTERVAL) * 1024 * 1024;
```

Comment:
the CTweak will automatically initialize itself, so you do not need this line.  Also, note that doing "* 1024 * 1024" will break things because the user can change this value at any time via RPC calls.  By multiplying it by 1 MiB here, you make pruneinterval the number of MiB if defined in bitcoin.conf or as an argument, but it remains PER BYTE if it is changed by RPC call, since it does not get multiplied there.

In summary, you need to multiply pruneInterval by MiB at the point where it is used.